src/Kernel/client/kernel.ts

```diff
@@ -147,52 +153,34 @@ class Kernel {
     }
 
     setupMeasurementHistogramDataListener() {
-        IPython.notebook.kernel.register_iopub_handler(
+        IPython.notebook.kernel.comm_manager.register_target<{state: DisplayableState}>(
```

Comment:
What about the other message handlers? Will those be moved to comms in a later PR?